packages/eslint-plugin-react-hooks/src/RulesOfHooks.js

```diff
@@ -324,6 +324,7 @@ export default {
           }
 
           cache.set(segment.id, paths);
+
```

Comment:
super nit: could we remove this whitespace change to keep blame history clean?

Reply:
Sure thing, done in [0f822bf](https://github.com/facebook/react/pull/28714/commits/0f822bfeb6eb40f6c42eec1ceaacbce154564aea)